lib/compile/csdl2openapi.js

```diff
@@ -453,12 +454,16 @@ module.exports.csdl2openapi = function (
      * @param {object} serversObject Input servers object
      * @return {Array} The list of servers
      */
-    function getServers(serviceRoot, serversObject) {
+    function getServers(serviceRoot, serversObject, csdl) {
         let servers;
 
         if (serversObject) {
             try {
                 servers = JSON.parse(serversObject);
+                // if csdl.$Version is 4.01 then protocol is rest if it is less than 4.01 then protocol is odata
```

Comment:
Can we add/update tests for this?

Reply:
added a test case to cover this scenario